.babelrc

```diff
@@ -1,3 +1,9 @@
 {
-  "presets": ["es2015", "stage-2", "stage-3"]
+  "env": {
```

Comment:
what does this change do? Doesn't seem like it would be necessary. Afaik we're not actually using babel anymore, so maybe this file will go away when we rebase?

Reply:
Yes, this was copied from my local repository for testing purposes. I will remove it.

Reply:
Babel was indeed removed (#1074), so just delete the file.

Reply:
i've removed it from this repository